elasticsearch/example/kustomization.yaml

```diff
@@ -38,4 +38,4 @@ patches:
         value: "4Gi"
       - op: replace
         path: /spec/template/spec/containers/0/resources/limits/memory
-        value: "8Gi"
+        value: "6Gi"
```

Comment:
Do we need to change this back, since I assume it was just for testing? I guess it doesn't really matter since it's just an example?

Reply:
> it doesn't really matter since it's just an example

I had this on my mind. Anyone using it is anyway expected to use resources as per their requirements. Both 8Gi and 6Gi are sensible example values